### PR TITLE
CASMINST-6574 CASMTRIAGE-6772 FAS new updates

### PR DESCRIPTION
**Pull request by mbuchmann-hpe**
# Description

Updated release notes with check_bios_firmware script issue.

Added issues in the known issues folder.

Added steps to Gigabyte updating M001 BMC/BIOS to check for completion.

Change to executable backup_smd_posgres.sh

PRs:
- CASMINST-6574
- CASMTRIAGE-6772

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Comment by mbuchmann-hpe** [comment]
The two checks that are failing are on a script that all I changed was the permissions on to make it executable, will not be making any changes to the script as that would require revalidation of that script.

**Review by jwlv — CHANGES_REQUESTED** [review]
Is the license check and shmft checks expected to fail?

**Comment by mbuchmann-hpe** [comment]
> Is the license check and shmft checks expected to fail?

I am ignoring the failings because the script permissions were the only thing that changed and I did not want to change the script any to avoid have to revalidate the script.

File: operations/firmware/Updating_Firmware_m001.md [hunk 1]
```diff
@@ -98,6 +95,19 @@ Use one of the following commands to find the model name for the node type in us
         -d '{"ImageURI":"http://ipaddressOfM001:8770/filename", "TransferProtocol":"HTTP", "UpdateComponent":"BMC"}'
     ```
 
+    Wait for the BMC to update and reboot
```
Comment (nrockershousen) [inline operations/firmware/Updating_Firmware_m001.md]:
```suggestion
    Wait for the BMC to update and reboot:
```

File: troubleshooting/known_issues/cray-tftp-upload.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+If you receive the following type of error please apply the workaround:
+
+```text
+Uploading file: curr.json
+error: source and destination are required
+Failed to upload curr.json - error code = 0
+```
+
+***Workaround:*** Edit the script `/usr/local/bin/cray-tftp-upload` Changing this line:
```
Comment (nrockershousen) [inline troubleshooting/known_issues/cray-tftp-upload.md]:
```suggestion
***Workaround:*** Edit the script `/usr/local/bin/cray-tftp-upload`, changing the following line:

```

File: troubleshooting/known_issues/cray-tftp-upload.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Known Issue: `cray-tftp-upload` errors
+
+* The `cray-tftp-upload` script errors out because of a change to the `ipxe` pods and the TFTP repository.
+* This error affects the `cray-upload-recovery-images` script.
+* This is expected to be fixed in CSM 1.5.1.
+
+## FIX
+
+If you receive the following type of error please apply the workaround:
```
Comment (nrockershousen) [inline troubleshooting/known_issues/cray-tftp-upload.md]:
```suggestion
If you receive the following type of error, then apply the workaround:

```

**Review by nrockershousen — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Review by jwlv — APPROVED** [review]

**Review by shunr-hpe — APPROVED** [review]

**Pull request merged**